nntrainer/tensor/blas_interface.cpp

```diff
@@ -838,4 +838,30 @@ unsigned int isamax(const unsigned int N, const float *X, const int incX) {
 #endif
 }
 
+void sine_transformation(const unsigned int N, float *X, float *Y,
```

Comment:
sine_transformation and cosine_transformation look very similar.
How about creating one conversion function and allowing it to be separated by input variables?
Wouldn't it be easy to support other transformations, such as tangent, later?

Reply:
Wouldn't it be more complex to add more hard-coded parameters for that?